### PR TITLE
Update einstein-overview.md

### PR DESCRIPTION
**Pull request by sbudhirajadoc**
We're fixing some minor doc bugs around whitespaces in this PR.

### What issues does this PR fix or reference?
#<Insert GitHub Issue>, @W-14155686@

### Functionality Before
minor doc bugs

### Functionality After
fixed minor doc bugs

### CODE REVIEW

**Review by ekapner — APPROVED** [review]

**Comment by sbudhirajadoc** [comment]
@gbockus-sf I regenerated response for the same prompt and updated the doc. Please review? Thank you!

File: docs/_articles/en/einstein/einstein-example-prompts.md [hunk 1]
```diff
@@ -79,15 +79,19 @@ return orderItems;
...
+trigger AccountTrigger on Account (before insert, before update, before delete, after insert, after update, after delete, after undelete) {
+    
+    if (Trigger.isBefore) {
+        
+    } else if (Trigger.isAfter) {
+        if (Trigger.isUpdate) {
+            List<Account> accountsWithChangedContacts = [SELECT Id, NumberOfContacts FROM Account WHERE Id IN :Trigger.newMap.keySet()];
+            for (Account a : accountsWithChangedContacts) {
+                a.NumberOfContacts = Trigger.newMap.get(a.Id).NumberOfContacts;
+            }
```
Comment (gbockus-sf) [inline docs/_articles/en/einstein/einstein-example-prompts.md:93]: I ran this through the scanner and had a few flagged
`	"message": "Sev3: Validate CRUD permission before SOQL/DML operation or enforce user mode",`
was the only one that really concerned me.  I'm going to have Chris take a look but I think we should add to the prompt request that `WITH USER_MODE` be added to the SOQL query.

Comment (gbockus-sf) [reply docs/_articles/en/einstein/einstein-example-prompts.md:93]: Chris says he will review by the end of the day :+1:

Comment (sbudhirajadoc) [reply docs/_articles/en/einstein/einstein-example-prompts.md]: @capeterson is it OK for us to merge this PR? Thanks!

File: docs/_articles/en/einstein/einstein-example-prompts.md [hunk 2]
```diff
@@ -76,19 +76,24 @@ return orderItems;
...
-        List<Contact> listaccContacts = [SELECT Id, Name, AccountId FROM Contact WHERE AccountId = :acc.Id];
-        if (acc.Contacts.size() < listaccContacts.size()) {
-            acc.addError('Not enough contacts');
-        }
-
+trigger AccountTrigger on Account (after update) {
+    List<Account> accountsWithContacts = [
+        SELECT Id, (SELECT Id FROM Contacts)
+        FROM Account
+        WITH USER_MODE
```
Comment (capeterson) [inline docs/_articles/en/einstein/einstein-example-prompts.md:86]: @sbudhirajadoc looks like we lose the WHERE clause on this from the prior version, so now it's querying *all* accounts, which doesn't match the prompt. 

As-is in this diff it tries to query ALL accounts and contacts which is just gonna blow out limits, so dangerous example.

Comment (sbudhirajadoc) [reply docs/_articles/en/einstein/einstein-example-prompts.md:87]: updated the response with chris's comment.

**Review by capeterson — APPROVED** [review]
lgtm

**Pull request merged**